test/kubernetes-services-test.el

```diff
@@ -85,7 +85,7 @@ Services (2)
 
 (ert-deftest kubernetes-services-test--sample-response ()
   (let ((state `((services . ,sample-get-services-response)
-                 (current-time . ,(date-to-time "2017-04-03 00:00Z")))))
```

Comment:
Can you please remove the changes? Looks like this is only failing for emacs master branch (ver 31.x). I made some changes to CI to not include 31.x test result in the job status.

Reply:
Done.